test/connection-pool.test.js

```diff
@@ -1,16 +1,13 @@
 var assert = require('assert');
 var Request = require('tedious').Request;
+var fs = require('fs');
 var ConnectionPool = require('../lib/connection-pool');
 
-var connectionConfig = {
-    userName: 'test',
-    password: 'test',
-    server: 'dev1',
-    options: {
-        appName: 'pool-test',
-        database: 'test'
-    }
-};
+function getConfig() {
+    var config = JSON.parse(fs.readFileSync(process.env.HOME + '/.tedious/test-connection-pool.json', 'utf8')).config;
```

Comment:
You don't even need  `fs` here. `require(process.env.HOME + '/.tedious/test-connection-pool.json', 'utf8').config` will return the same result while being less noisy.